Lenovo/Z5s/res/values/notch.xml

```diff
@@ -17,9 +17,7 @@
 */
 -->
 <resources>
-    <dimen name="rounded_corner_radius">30dip</dimen>
-    <dimen name="status_bar_height">30.0dip</dimen>
-    <dimen name="status_bar_height_landscape">30.0dip</dimen>
-    <dimen name="status_bar_height_portrait">30.0dip</dimen>
+    <dimen name="rounded_corner_radius">30dp</dimen>
```

Comment:
Is it a device with 480 dpi by default? Then that would translate to 90px instead, as rounded corner radius is physical.

Reply:
Ye, thank you,i have modified